electron/install.sh

```diff
@@ -81,6 +81,9 @@ fi
   -m "$ARGV_NPM_VERSION" \
   -l electron
 
+"$HERE/../shared/sanity-checks.sh" \
+  -b "$ARGV_BASE_DIRECTORY"
+
```

Comment:
@thundron this needs the NPM_VERSION as an argument as well; why do we need a second run of sanity-checks?

Reply:
@nazrhom We need it in case the second run of `npm i` modifies the shrinkwrap again. It happened to me while developing, because sometimes the `postshrinkwrap` step after modules installation, for *some reason*, happened before the actual modules installation/shrinkwrap step, so by running again the sanity-checks we also run `npm shrinkwrap --dev` which fixes the shrinkwrap

Reply:
@nazrhom Because after the second run of `npm install`, for *some reason*, sometimes the `postshrinkwrap` step runs after optional dependencies installation (that in etcher cleans the shrinkwrap with https://github.com/balena-io/etcher/pull/2768) so by running again the sanity-checks we also run `npm shrinkwrap --dev` which fixes the shrinkwrap with the `clean-shrinkwrap.js` script (local to the project for now until we figure out why we need to run install two times and also why we can't have optional dependencies in the shrinkwrap for other OSes)

Reply:
@nazrhom Added the NPM_VERSION parameter on the second call too